tests/test_functional.py

```diff
@@ -136,10 +126,10 @@ def test_dynamic_blockwise_quantization(self, device, dtype, nested, blocksize,
         abserr = sum(diffs) / len(diffs)
         relerr = sum(reldiffs) / len(reldiffs)
         if signed:
-            assert abserr < 0.0035
+            assert abserr < 0.0036
             assert relerr < 0.015
         else:
-            assert abserr < 0.00175
+            assert abserr < 0.0023
```

Comment:
It's better if we didn't change this just for CPU/XPU support; if we need to let's adjust based on `device == "cpu"` and maybe in combination with `dtype` and `blocksize` for now.

Reply:
fixed